src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -426,17 +426,11 @@
   <data name="InvalidFilename.Text" xml:space="preserve">
     <value>The item name specified is invalid</value>
   </data>
-  <data name="ItemSelected.Text" xml:space="preserve">
-    <value>item selected</value>
+  <data name="pItemsSelected" xml:space="preserve">
```

Comment:
I prefer not having `.` in the string name

Reply:
Actually I really don't like this `p`

Reply:
Do you prefer `ItemsSelected` and adding a comment to indicate that it's plural?

Reply:
Sure

Reply:
So should I remove the `p` prefix and add a comment?
What should the comment look like?

```xml
<data name="ItemsSelected" xml:space="preserve">
  <value>{0, plural, one {# item selected} other {# items selected}}</value>
  <comment>ICU format for plurals, more information here: https://support.crowdin.com/icu-message-syntax/#plural</comment>
</data>
<data name="ItemsCount" xml:space="preserve">
  <value>{0, plural, one {item} other {items}}</value>
  <comment>ICU format for plurals, more information here: https://support.crowdin.com/icu-message-syntax/#plural</comment>
</data>
```

Reply:
Yes, sorry for the back and forth on this!

Reply:
Okay, I'm on it, I like both the dot option and adding a comment, so we'll use the standard title and comment

But we have to choose a different name for `ItemsCount` because we can't use
![image](https://github.com/files-community/Files/assets/84145589/02d9c01b-66ed-4b52-aa5d-66bbc2bbf204)

Reply:
I forgot what the error represents I’ve seen in UWP.
It’s maybe used in MUXC.

What about Items, ItemsCountInfrectable?

Reply:
maybe `Items` would be good to use

now used for only this
![image](https://github.com/files-community/Files/assets/84145589/838a8ad0-0448-4684-a13a-6f251a6c0edb)

Reply:
And for `ItemsSelected` ?
There are same problem

Reply:
Maybe invert to `SelectedItems`?

Reply:
Checking